Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -182,6 +182,9 @@ function mac_process_network_entitlements()
         plistbuddy Add :com.apple.private.webkit.adattributiond bool YES
         plistbuddy Add :com.apple.private.webkit.webpush bool YES
         plistbuddy Add :com.apple.developer.hardened-process bool YES
+        plistbuddy Add :com.apple.private.get-system-corpse bool YES
+        plistbuddy Add :com.apple.security.exception.mach-lookup.global-name array
+        plistbuddy Add :com.apple.security.exception.mach-lookup.global-name:0 string com.apple.ReportMemoryException
```

Comment:
I think it would be good to add these to the Networking process sandbox, instead of using an entitlement. That way, we have the allowed mach lookups gathered in one place. It would also be good if we could only allow this on internal builds.